src/client.ts

```diff
@@ -96,6 +103,34 @@ export class ZerionAPI implements iZerionAPI {
     );
     return data;
   }
+
+  async getNativeTokens(params?: {
+    useStatic?: boolean;
+    supportedChains?: number[];
```

Comment:
You might want to give the chains list to this function. since the caller already has it. This function shouldn't be responsible for fetching chains.